libmetal/README

```diff
@@ -0,0 +1,39 @@
+libmetal
+#####################
+
```

Comment:
Should this README look like the one in open-map?

Reply:
Yes it does.  I made updates on my PC but seems that I don't push the good version